src/astro/__init__.py

```diff
@@ -30,3 +34,11 @@ def get_provider_info() -> dict:
         "hook-class-names": [],
         "extra-links": [],
     }
+
+
+if not conf.getboolean(section="core", key="enable_xcom_pickling"):
+    raise OSError(
+    raise OSError(
+        "AIRFLOW__CORE__ENABLE_XCOM_PICKLING environment variable needs to be set to True or enable_xcom_pickling=true "
+        "in airflow.cfg before importing astro-sdk-python."
```

Comment:
It may be worth confirming with @mikeshwe this message..! Due to timezone, I think we can merge the PR as it is, and improve it after his feedback

Reply:
There are also multiple ways of setting configuration including secrets backend as documented in https://airflow.apache.org/docs/apache-airflow/stable/howto/set-config.html

So probably just worth simplifying that set `enable_xcom_pickling` airflow config to true. And if you'd like to be more helpful to user, then say " One of the simplest ways is by adding the environment variable AIRFLOW__CORE__ENABLE_XCOM_PICKLING=True`